client/src/InfoBase/App.js

```diff
@@ -48,6 +48,10 @@ const BudgetMeasuresRoute = retrying_react_lazy(() =>
 const About = retrying_react_lazy(() =>
   import(/* webpackChunkName: "About" */ "../about/about.js")
 );
+
+const Contact = retrying_react_lazy(() => 
+  import("../contact/contact.js")
```

Comment:
Oh, please add a "magic comment" `/* webpackChunkName: "Contact" */` inside this import. It's just a bit of metadata that webpack will look for and use to give more useful names to the output of the build process.